docs/dev/handbook/cluster_creation.md

```diff
@@ -0,0 +1,187 @@
+
+Note: in theory `make docker-all` will trigger all `docker-build`, `docker-publish` and `kbld-image-replace`. But
+locally for me, it only runs the first target with this version
+of [Makefile](https://github.com/vmware-tanzu/tanzu-framework/blob/0f572c2ee0d209f538d33b3c5951a664f4076a82/Makefile#L740)
+.
+
+After that, run the local registry. Then, build the package and repo bundles for a management clusters.
+
+```shell
+make local-registry && make package-push-bundles-repo PACKAGE_REPOSITORY=management
+```
+
+## Link clusters to the package repository and install packages
+### Spin up a kind cluster as the management cluster
+
+Create kind cluster with CAPx providers and kapp-controller installed. Since this utility script install multiple
+infrastructure providers,
+environment variables should be exported such as `AWS_B64ENCODED_CREDENTIALS`, `VSPHERE_PASSWORD`, `VSPHERE_USERNAME`.
+
+```shell
+export AWS_B64ENCODED_CREDENTIALS=<encoded aws credentials>
+./hack/kind/deploy_kind_with_capi_and_kapp.sh
+```
+
+Login to the kind cluster with tanzu CLI.
+
+```shell
+tanzu login --kubeconfig '${HOME}/.kube/config' --context kind-test-cluster --name kind-test-cluster
+```
+
+### Connect the kind cluster to the package repository
+
+Add the `management` repository with packages we just built to the management cluster,
+
+```shell
+export PACKAGE_REPO_URL=gcr.io/my-project-1527816345739/tkg/management/management@sha256:0bd76eb7c622fa32cbb851b30d691dc797105445f5e42ec8dd2dc41b34366fdf
+tanzu package repository update management --url ${PACKAGE_REPO_URL} --create
+```
+
+Ensure that the `management` repository has been reconciled successfully.
+
+```shell
+$ tanzu package repository list
+  NAME        REPOSITORY                                                 TAG                                                                      STATUS               DETAILS
+  management  gcr.io/my-project-1527816345739/tkg/management/management  sha256:0bd76eb7c622fa32cbb851b30d691dc797105445f5e42ec8dd2dc41b34366fdf  Reconcile succeeded
+```
+
+At this point, it is expected to see all packages that are available to install.
+```shell
+$ kubectl get package -A 
+NAMESPACE   NAME                                                      PACKAGEMETADATA NAME                           VERSION      AGE
+default     addons-manager.tanzu.vmware.com.0.23.0-dev                addons-manager.tanzu.vmware.com                0.23.0-dev   10s
+default     cliplugins.tanzu.vmware.com.0.23.0-dev                    cliplugins.tanzu.vmware.com                    0.23.0-dev   10
+...
+```
+
+In order to create a cluster with cluster-api-aws, we need to install the `tkg-clusterclass-aws` package first. It is possible to hit
+package reconcile error if k8s version is 1.24 or above with a older version of kapp-controller such as v0.31.0 (Troubleshoot 3). Let's grep it first.
+
+```shell
+$  k get packages -A | grep tkg-clusterclass-aws
```

Comment:
NIT: I use the `k` alias as well but it might be better for users who don't to use `kubectl` instead

Reply:
yep will do :)